src/main/java/seedu/address/model/internship/Date.java

```diff
@@ -32,7 +34,17 @@ public Date(String date) {
      * @returns true if a given string is a valid date string, else returns false.
      */
     public static boolean isValidDate(String test) {
-        return test.matches(VALIDATION_REGEX);
+        if (!test.matches(VALIDATION_REGEX)) {
+            return false;
+        };
```

Comment:
I think there is an additional semi-colon here. This causes a checkstyle violation:
![image](https://user-images.githubusercontent.com/86542359/223313347-116bba81-dd3b-4e58-abc0-2c8061367a09.png)

As a tip, you could double press `ctrl` and run `gradlew checkstyleMain checkstyleTest` to ensure that there is no coding style violations.

Reply:
really helpful tip! thanks Eugene! Have done the changes